problem15_missingNumber.java

```diff
@@ -0,0 +1,82 @@
+package CompetitiveCoding;
+
+import java.util.HashSet;
+
+public class problem15_missingNumber {
+
+    /*
+    Using HashSet ---->
+        Time Complexity : O(n)
+        Space Complexity: O(n) I guess, I am not sure about this tho as
+                               I am checking if the set contains the missing value or not.
+     */
+    private static int missingElementHashSet(int[] arr) {
+        HashSet<Integer> set = new HashSet<>();
+        for(int a: arr) {
+            set.add(a);
+        }
+        int finalVal = arr.length + 1;
+
+        for(int i = 0 ; i < finalVal ; i++) {
+            if(!set.contains(i))
+                return i;
+        }
+        return -1;
+    }
+
+    /*

```

Comment:
Good Job, make sure to watch the video shared on the S30 portal to get a better understanding